llvm/test/CodeGen/AArch64/sve-fixed-length-sdiv-pow2.ll

```diff
@@ -383,4 +383,19 @@ define void @sdiv_v32i64(ptr %a) vscale_range(16,0) #0 {
   ret void
 }
 
+define i32 @sdiv_int(i32 %begin, i32 %first) #0 {
```

Comment:
This test isn't actually testing fixed-length vector code so I think a more appropriate place for the test would be something like `llvm/test/CodeGen/AArch64/sdivpow2.ll` where we test the scalar optimisation. Also, I think the test can be made simpler, i.e. something like

```
define i32 @test9(i32 %x) #0 {
  %div = sdiv i32 %x, 8
  ret i32 %div
}

attributes #0 = { "target-features"="+sve" vscale_range(2,2) }
```

Reply:
hi @david-arm the issue exist only iff there is option **-msve-vector-bits=256** in the command line, but it doesn't have that option in test **llvm/test/CodeGen/AArch64/sdivpow2.ll**, so would it be better to add a new separate test file ?

Reply:
Done, thanks for your suggestion.